specs/capri/OpenAPI_Design.md

```diff
@@ -0,0 +1,160 @@
+# VMware Ecosystem OpenAPI
+
+**Author(s)**: [zengyingzhe](https://github.com/zengyingzhe)
+This documentation serves as the design spec for OpenAPI.
```

Comment:
Please move this line to `## Summary` section

Reply:
Please specify "VMware" in front of "OpenAPI".  Otherwise it is too general.

Reply:
Done